gui/src/main.rs

```diff
@@ -37,7 +38,8 @@ impl Default for GuiConfig {
             active_window: ActiveWindow::REQUEST,
             request_window_mode: RequestWindowMode::BODY,
             selected_http_method: HttpMethod::GET,
-            url: String::from("https://localhost:3000"),
+            url: String::from("https://httpbin.org/json"),
```

Comment:
This httpbin site I came across is pretty sweet for testing this! there are routes for each method like /get, /post, etc.